posts/2021-12-06-survey-launch.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Launching the 2021 State of Rust Survey"
+author: The Rust Community Team
+description: "Hearing from you about the sixth year of Rust"
+---
+
+It's that time again! Time for us to take a look at who the Rust community is composed of, how the Rust project is doing, and how we can improve the Rust programming experience. The Rust Community Team is pleased to announce our [2021 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses, and establish development priorities for the future.
+
+Completing this survey should take about 10–30 minutes and is anonymous. We will be accepting submissions for the next two weeks (until the 6th of December), and we will write up our findings afterwards to [blog.rust-lang.org]. You can also check out [last year’s results][2020 survey].
+
+- [English]
+- [Simplified Chinese]
+- [Traditional Chinese]
+- [French]
+- [German]
+- [Japanese]
+- [Korean]
+- [Portuguese]
+- [Russian]
+- [Spanish]
+
+(If you speak multiple languages, please pick one)
+
+Please help us spread the word by sharing the survey link on your social network feeds, at meetups, around your office, and in other communities.
+
+If you have any questions, please see our [frequently asked questions] or email the Rust Community team at [community-team@rust-lang.org].
+
+Finally, we wanted to thank everyone who helped develop, polish, and test the survey. In particular, we'd like to thank all of the volunteers who worked to provide all of the translations available this year and who will help to translate the results.
+
+[blog.rust-lang.org]: https://blog.rust-lang.org
+[frequently asked questions]: https://github.com/rust-lang/surveys/blob/main/documents/Community-Survey-FAQ.md
+[community-team@rust-lang.org]: mailto:community-team@rust-lang.org
+[2020 survey]: https://blog.rust-lang.org/2020/12/16/rust-survey-2020.html
+
+[survey]: https://surveyhero.com/c/jzesmzph
+[English]: https://surveyhero.com/c/dauv4jcd
+[Portuguese]: https://surveyhero.com/c/bhdqz3hj
+[Simplified Chinese]: https://surveyhero.com/c/7rnfv4cf
+[French]: https://surveyhero.com/c/y7hb34nm
+[Korean]: https://surveyhero.com/c/m3bi3nad
+[Spanish]: https://surveyhero.com/c/he7yvafu
+[Russian]: https://surveyhero.com/c/xix7rf7d
+[Traditional Chinese]: https://surveyhero.com/c/h7guatdt
+[German]: https://surveyhero.com/c/pq43azn4
```

Comment:
![image](https://user-images.githubusercontent.com/2129/144830166-79ba98a2-a6e7-4b52-988b-b5f4d823aae3.png)

For the German version `Rust Programmiersprache` is not properly linked, instead the url is visible after it.

Reply:
I'll fix that - for anyone else looking here who has feedback on the survey's themselves, you can report them [here](https://rust-lang.zulipchat.com/#narrow/stream/294169-t-community.2Frust-survey-2021).